### PR TITLE
fix: remove unnecessary movement of eval logits to cpu

### PR DESCRIPTION
**Pull request by NanoCode012**
# Description

Edit: See https://github.com/axolotl-ai-cloud/axolotl/pull/2824#issuecomment-3002851516

---

There has been reports about eval taking more vram than training. I suspect it's this config which we set to grad accu. Would love some tests for this!

This config does not seem to be running mini-batches as we think?

https://huggingface.co/docs/transformers/main_classes/trainer#transformers.TrainingArguments.eval_accumulation_steps

## Summary by CodeRabbit

- **Refactor**
  - Removed explicit setting of the "eval_accumulation_steps" parameter in training configuration.

### CODE REVIEW

**Comment by NanoCode012** [comment]
From reading the trainer evaluation loop, I don't think we need this config. The VRAM consumed by just an eval pass is very small compared to training, so this config would just cause unnecessary gpu->cpu movement.

However, it should not be the cause for increased vram usage as initially stated.
